ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/FcsErrors.xml

```diff
@@ -320,4 +320,15 @@
     <QuickFix>ReplaceWithAbbreviatedTypeFix</QuickFix>
   </Error>
 
+  <Error staticGroup="FSharpErrors" name="RuntimeCoercionSourceSealed">
+    <Parameter type="IFSharpExpression" name="expr"/>
```

Comment:
According to the quick fix code, shouldn't it be `ITypeTestExpr` instead?

Reply:
Any update here?

Reply:
I think this error applies not only to ITypeTestExpr.